backend/src/main/java/com/huellapositiva/application/dto/ChangeStatusVolunteerDto.java

```diff
@@ -0,0 +1,15 @@
+package com.huellapositiva.application.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class ChangeStatusVolunteerDto {
+
+    private String idProposal;
+    private String idVolunteer;
+    private boolean confirmed;
```

Comment:
I am not sure that a boolean is enough. I can see we have three states:
Volunteer joins proposal -> status - pending
Volunteer is rejected -> status rejected
Volunteer is confirmed -> status confirmed